src/igvfd/searches/configs/workflow.py

```diff
@@ -10,15 +10,6 @@ def workflow():
             'uniform_pipeline': {
                 'title': 'Uniform Pipeline'
             },
-            'analysis_steps.analysis_step_versions.software_versions.software.title': {
-                'title': 'Software',
-            },
```

Comment:
this should be replaced with analysis_step_versions.software_versions.software.title not removed

Reply:
Fixed, thanks.